scripts/twister

```diff
@@ -685,6 +686,10 @@ structure in the main Zephyr tree: boards/<arch>/<board_name>/""")
         help="Execute time-consuming test cases that have been marked "
              "as 'slow' in testcase.yaml. Normally these are only built.")
 
+    parser.add_argument(
+        "--seed", type=int, default=random.randrange(8192),
```

Comment:
By default twister should not switch random seeds for the underlaying tests. It should be a conscious choice by the user to do so. Otherwise a) people will be baffled when tests fail in one run but not the next. b) it adds flakiness to CI where tests fail only some times.

Reply:
@yperess - any concerns with pushing the seed randomness onto the user?

Reply:
I don't mind at all, Zephyr's CI belongs to them. I personally am in favor of having it done that way this way we can keep finding issues without someone having to manually do it at home, but I 100% understand if Zephyr doesn't want to do it. For what it's worth, Google does this and it's OK, we just say if a failure happens in a PR that doesn't touch it you should just re-run the submit task. Another bot will look at which tests fail and will mark a test as flaky.

Reply:
I think we should disable this by default. it may confuse users to reproduce issues.